src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseDetailsPageUiTest.java

```diff
@@ -174,7 +174,7 @@ public void testRemindAction() {
         
         //Alice is already registered
         StudentAttributes alice = testData.students.get("CCDetailsUiT.alice.tmms@CCDetailsUiT.CS2104");
-        String alicePassword = TestProperties.inst().TEST_STUDENT2_PASSWORD;
+        String alicePassword = TestProperties.inst().TEST_STUDENT1_PASSWORD;
```

Comment:
It's used in more places than just this particular part of the test. I guess better leave it to first timer/contributor to fix the entire file for good.